device/sycl/src/seeding/triplet_finding.sycl

```diff
@@ -217,9 +218,10 @@ class TripletFind {
         }
         // Calculate the number of triplets per "block" with reducing sum
         // technique
-        ::sycl::group_barrier(workGroup);
+        // ::sycl::group_barrier(workGroup);
+        item.barrier();
```

Comment:
Maybe, adding this would make the synchronization pretty much the same as before?
```cpp
item.barrier(::sycl::access::fence_space::local_space);
```

Reply:
I had the same idea, but after looking at the oneAPI headers for a bit, I wasn't entirely sure that this would be a 1-to-1 replacement. So I went with this safer-looking global barrier in this PR...